sdks/python/apache_beam/io/google_cloud_platform/datastore/v1/helper.py

```diff
@@ -220,7 +227,7 @@ class QueryIterator(object):
 
   Entities are read in batches. Retries on failures.
   """
-  _NOT_FINISHED = query_pb2.QueryResultBatch.NOT_FINISHED
+  _NOT_FINISHED = QUERY_NOT_FINISHED
```

Comment:
Do we need this here? I think we only need to guard against module inits. Using classes with missing imports should fail.